src/factoryItems/ArticleFactory.js

```diff
@@ -0,0 +1,38 @@
+import { faker } from '@faker-js/faker';
+
+export class ArticleFactory {
+
+  generateArticle(tagNumber = 0) {
+
+    const tags = [];
+    for (let i = 0; i < tagNumber; i++) {
+      tags.push(this.generateTag());
+    }
+    const title = this.generateTitle();
+    const description = this.generateDescription();
+    const text = this.generateText();
+
+    return {
+      title,
+      description,
+      text,
+      tags,
+    };
+  }
+
+  generateTitle() {
+    return faker.lorem.words(5);
+  }
+
+  generateDescription() {
+    return faker.lorem.sentence(4);
```

Comment:
you can create helper function in a separate file to generate sentences with a required lengs and call it here, e.g.:

```js
export function generateSentenceWithLengs(number) {
  return faker.lorem.sentence(number);
} 

export function generateWords(number = 1) {
  return faker.lorem.word(number);
} 
```